scripts/west_commands/genboard/config.yml

```diff
@@ -0,0 +1,93 @@
+products:
+  - series: nrf52
+    socs:
+      - name: nrf52805
+        variants:
```

Comment:
with HWMv2 in mind, then I don't think `variants` the best key name to use here.

Reply:
not sure what to pick :-) will ask internally if the `qkaa` et al. thing has an official name

Reply:
perhaps `package` ?

https://infocenter.nordicsemi.com/index.jsp?topic=%2Fcomp_matrix_nrf52840%2FCOMP%2Fnrf52840%2FnRF52840_ic_revision_overview.html&cp=3_0_3_0

Reply:
problem is that some packages also imply different memory setups at least in 1 case

Reply:
`package_variant` seems the official name will update then.